src/components/constants.ts

```diff
@@ -1,2 +1,10 @@
 export const PLAYER_MINIMUM_GAMES_REQUIRED = 10;
-export const COMMANDER_MINIMUM_GAMES_REQUIRED = 5;
\ No newline at end of file
+export const COMMANDER_MINIMUM_GAMES_REQUIRED = 5;
+
+export const MTG_COLORS: { name: string; id: string; rgb: string }[] = [
+    { name: "Black", id: "B", rgb: "rgb(166, 159, 157)" },

```

Comment:
nit: we can probably make this an actual domain type instead of an in-line type declaration. not required for this PR, but something we'll want to do sooner than later.